exonum-java-binding-cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/ApiController.java

```diff
@@ -83,6 +83,9 @@ void mountApi(Router router) {
   }
 
   private void submitTransaction(RoutingContext rc) {
+    checkRequestContentType(rc, "application/octet-stream");
```

Comment:
Is it required for all Exonum Services implementation ? Should be specified.

Reply:
Not sure I understand what you mean. This code is in a particular service implementation, in a particular endpoint. It is not a part of the framework.

Reply:
I think @DENMROOT meant that we might need to fix cryptocurrency frontend as well.

Reply:
We won't integrate that till front is updated: https://jira.bf.local/browse/ECR-2650?focusedCommentId=30419&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-30419